index.html

```diff
@@ -6,6 +6,392 @@
 	<link rel="stylesheet" type="text/css" href="index.css">
 </head>
 <body>
-	
+
+	<!-- intro -->
+	<header class="row">
+
+		<img src="assets/yellow-pill.svg" alt="yellow pill">
+		<img src="assets/more-pills.svg" alt="a blue pill, a purple pill, and a yellow pill">
+		
+		<span class="subTitle">A Viget Exploration</span>
+		<h1>
+			<img src="assets/title.svg" alt="Improving Medical Adherence">	
+		</h1>
+		
+		<img src="assets/pills.svg" alt="a blue pill and yellow pill">
+		<img src="assets/purple-pills.svg" alt="two purple pills">
+
+		<!-- yellow-pill.svg is top left -->
+		<!-- more-pills.svg is top right -->
+		<!-- pills.svg is bottom left -->
+		<!-- purple-pills.svg is bottom right -->
+
+		<section class="whiteBox">
```

Comment:
Could you please give an example of how to rethink this project so that it adheres to BEM styling/naming conventions?